examples/ingress.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: prj-example-1-frontend
+  namespace: prj-example-1
+  annotations:
+    konghq.com/strip-path: "true"
+spec:
+  ingressClassName: kong
+  rules:
+  - http:
+      paths:
+      - path: /prj-example-1/frontend
```

Comment:
I think we should use the host "foo.dmtr.host" since our `demeter.runinstance=foo` vs path

Reply:
Sorry, I wasn't clear. I meant instead of a path.